master/getting-started/openstack/installation/redhat.md

```diff
@@ -313,9 +321,9 @@ On each compute node, perform the following steps:
     reboot.
 
     ```
-    service bird restart
-    service bird6 restart
-    chkconfig bird on
+    service bird restart \
```

Comment:
Same here - all of these won't work the way they are written now. I think you should just remove the trailing `\`.